iphone/Maps/UI/PlacePage/PlacePageViewController.swift

```diff
@@ -75,7 +75,7 @@ final class PlacePageScrollView: UIScrollView {
     actionBarContainerView.layer.masksToBounds = true
     actionBarContainerView.layer.maskedCorners = [.layerMinXMaxYCorner, .layerMaxXMaxYCorner]
 
-    if previousTraitCollection == nil {
+    if #available(iOS 13.0, *), previousTraitCollection == nil {
```

Comment:
Thanks! Please add short comments referencing the fixed issue to make it clear for future code readers.